source/_components/sensor.recollect_waste.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Recollect Waste Sensor
+description: "Instructions on how to set up Recollect Waste sensor within Home Assistant."
+date: 2019-01-14 13:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: recollect-waste.png
+ha_category: Sensor
+ha_release: 0.85
+ha_iot_class: "Cloud Polling"
+---
+
+The `Recollect Waste` platform allows you to track the next scheduled waste pickup and what type of waste from [Recollect](https://recollect.net/solutions/waste/). To use this sensor your city's waste company must be Recollect and you will need to find your place_id and service_id.
+
+1. In Chrome open developer tools and go to network tab.
+2. Go to your city's Recollect collection calendar.
+3. Search for and select your address in the UI.
+4. Watch for a request that looks like
+
+   ht<span>tps://api.recollect.net/api/places/**(place_id)**/services/**(service_id)**/events?nomerge ...
+
+5. Use the place_id and service_id when configuring the sensor.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: recollect_waste
+    place_id: YOUR_PLACE_ID
+    service_id: YOUR_SERVICE_ID
+```
+
+Configuration options for the Recollect Waste Sensor:
```

Comment:
Please us configuration tags. For more information:

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration